From bc7faea25b5dfc9ba43f17a31479fde273e49f67 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 15 Sep 2014 05:32:50 -0700
Subject: [PATCH] packages.py: update for kernel-image

Use kernel-image to locate the bzImage.

Inappropriate [WR-installer specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/packages.py |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pyanaconda/packages.py b/pyanaconda/packages.py
index 0b0bf63..c60f661 100644
--- a/pyanaconda/packages.py
+++ b/pyanaconda/packages.py
@@ -261,9 +261,9 @@ def rpmKernelVersionList(rootPath = "/"):
         return ""
 
     def get_tag(header):
-        if header['name'] == "kernel":
+        if header['name'] == "kernel-image":
             return "base"
-        elif header['name'].startswith("kernel-"):
+        elif header['name'].startswith("kernel-image"):
             return header['name'][7:]
         return ""
 
@@ -272,7 +272,7 @@ def rpmKernelVersionList(rootPath = "/"):
     iutil.resetRpmDb(rootPath)
     ts = rpm.TransactionSet(rootPath)
 
-    mi = ts.dbMatch('provides', 'kernel-image-base')
+    mi = ts.dbMatch('provides', 'kernel-image')
     for h in mi:
         v = get_version(h)
         tag = get_tag(h)
-- 
1.7.9.5

